### PR TITLE
add all the notecards

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds all the notecards to the Kitchensink

### CODE REVIEW

File: files/en-us/mdn/kitchensink/index.html [hunk 1]
```diff
@@ -361,3 +361,149 @@ <h2 id="Types">Types</h2>
...
+
+{{ B2GOnlyHeader2("privileged") }}
+
+<h3>B2GOnlyHeader2("installed")</h3>
+
+{{ B2GOnlyHeader2("installed") }}
+
+<h3>B2GOnlyHeader2("certified")</h3>
+
+{{ B2GOnlyHeader2("certified") }}
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:385]: All of these `B2G` ones should probably never ever be used anyway. 
But it's extremely possible that there's some **archived** page that still has:
```html
<div class="blockIndicator warning">
    <p style="text-align:center">This API is available on <a href="/en-US/docs/Mozilla/Firefox_OS">Firefox OS</a> for <a href="/en-US/docs/Mozilla/Firefox_OS/Security/Application_security#App_Types">web content or higher privileged applications</a>.</p>
</div>
```
in its rendered out HTML. 
That HTML won't ever change. So you'll need to have a SCSS definition for `.blockIndicator.warning` still in Yari.

Comment (peterbe) [reply files/en-us/mdn/kitchensink/index.html:385]: It makes me wonder, should there be a CSS file specifically for archived content?

File: files/en-us/mdn/kitchensink/index.html [hunk 2]
```diff
@@ -361,3 +361,149 @@ <h2 id="Types">Types</h2>
 {{Non-standard_Header}}
 {{Obsolete_Header}}
 {{Deprecated_Header}}
+
+<h2 id="notecards">Notecards</h2>
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:365]:
```suggestion
<h2>Notecards</h2>
```

**Review by peterbe — COMMENTED** [review]
I'm just not sure that this is the right thing to do. 

When I added macros to the kitchensink in the first place, I did so by analyzing all the *active* content. Sure, I probably missed some KS macro but when I did it I did it by sorting by how frequently they were used. And I worked from the last down. So if I missed some macro, it was probably used on 1-3 pages at most. 

I'm not sure we should bother with all these old notecard related macros. 
For example, the `B2GOnlyHeader2` macro is dead and if you edit `macros/B2GOnlyHeader2.ejs` it won't ever render out to something you can or should control. 

The Kitchensink is called "kitchen sink" but I'm still wondering if we should be somewhat careful with what we put into it. 
If you genuinely want trigger a render of every possible legacy macro, you could just do something like this PR but just leave it in a branch and discard when you're done.

**Comment by peterbe** [comment]
Another comment I have is that the Kitchensink should break as we discard some old macros that are never ever used in the *active* content.

File: files/en-us/mdn/kitchensink/index.html [hunk 3]
```diff
@@ -361,3 +361,149 @@ <h2 id="Types">Types</h2>
...
+
+{{ B2GOnlyHeader2("installed") }}
+
+<h3>B2GOnlyHeader2("certified")</h3>
+
+{{ B2GOnlyHeader2("certified") }}
+
+<h3>CompatibilityTable()</h3>
+
+{{ CompatibilityTable() }}
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:389]: For example, 
```
content/files/en-us  main ✔
▶ rg CompatibilityTable

```
means that it's never used by any of the (en-US) pages that render KumaScript. 
So this one we can ignore.

File: files/en-us/mdn/kitchensink/index.html [hunk 4]
```diff
@@ -361,3 +361,149 @@ <h2 id="Types">Types</h2>
...
+
+{{ DeprecatedGeneric("inline") }}
+
+<h3>DeprecatedGeneric("header")</h3>
+
+{{ DeprecatedGeneric("header") }}
+
+<h3>DeprecatedGeneric("method")</h3>
+
+{{ DeprecatedGeneric("method") }}
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:401]: ditto
```
content/files/en-us  main ✔
▶ rg DeprecatedGeneric

```

File: files/en-us/mdn/kitchensink/index.html [hunk 5]
```diff
@@ -361,3 +361,149 @@ <h2 id="Types">Types</h2>
 {{Non-standard_Header}}
 {{Obsolete_Header}}
 {{Deprecated_Header}}
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:363]: Aren't these notecards too? If so, they ought to sit underneath the `<h2>Notecards</h2>` header.

**Comment by schalkneethling** [comment]
Your comment at https://github.com/mdn/content/pull/93#pullrequestreview-527377038 clears things up for me. With that, I reckon I am going to use the branch for testing purposes and kill it afterward. Thanks @peterbe

**Pull request closed without merging**